ProjectSidewalk/SidewalkWebpage
# Skipped tutorial does not count as mission completed

Resolves #2296

Due to the selection we were using to accumulate mission completes, we were counting skipped tutorials as completed missions. 

I added a boolean to that method to specify whether we want to include skipped missions or not, and passed in the correct parameters from different places to adjust whether we count skipped missions or not. It needs to be differentiated whether we count skipped missions or not because of dependencies in whether we show the tutorial again or not, since this method is used in more places than just the user dashboard.

##### Before/After screenshots (if applicable)

##### Testing instructions
1. create a new account
2. go to "Start Exploring"
3. skip the tutorial
4. go to the user dashboard
5. verify missions completed is 0
